tools/nl/detection_evals/golden_curation/generate_vars.py

```diff
@@ -0,0 +1,370 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+from dataclasses import dataclass
+import json
+import os
+from typing import Dict, List, Set, Tuple
+
+from absl import app
+from absl import flags
+import requests
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('input_folder', 'input',
+                    'The input directory of files with queries to get svs for')
+flags.DEFINE_string(
+    'output_folder', 'output',
+    'The output directory to save results of getting svs for each query')
+flags.DEFINE_string(
+    'topic_cache', 'topic_cache.json',
+    'file path to a copy of the topic cache at server/config/nl_page/topic_cache.json'
+)
+
+_MAX_TOPICS_DEPTH = 3
```

Comment:
How did you choose 3 for this?

Reply:
Because the stat vars of a peer group will be at atleast level 2 so I wanted one more for good measure